forge/forge/op/eval/forge/eltwise_nary.py

```diff
@@ -144,7 +144,12 @@ def get_eltwise_shape_and_broadcast():
         axis = attr[0]
 
         output_shape = list(ops[0])
-        output_shape.insert(axis, len(ops))
+        if axis == -1:
+            output_shape.append(len(ops))
+        elif axis < 0:
+            output_shape.insert(axis + 1, len(ops))
```

Comment:
can we put elif axis < -1 just to be precise. Also can you add just one line of comment why we use + 1 to handle inserting new axis. This is just to help someone that lands in this part of the code and doesn't have the context...

Reply:
Sure @dgolubovicTT , I will make the changes.